app/src/main/java/com/infomaniak/drive/ApplicationMain.kt

```diff
@@ -54,13 +61,18 @@ import io.sentry.SentryEvent
 import io.sentry.SentryOptions
 import io.sentry.android.core.SentryAndroid
 import io.sentry.android.core.SentryAndroidOptions
-import kotlinx.coroutines.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
 import okhttp3.Interceptor
 import okhttp3.OkHttpClient
-import org.eclipse.paho.client.mqttv3.*
+import org.matomo.sdk.Tracker
+import org.matomo.sdk.extra.MatomoApplication
 import java.util.*
 
-class ApplicationMain : Application(), ImageLoaderFactory {
+
+class ApplicationMain : MatomoApplication(), ImageLoaderFactory {
```

Comment:
Is it possible to do it another way?

It's generally a bad idea to use another Application class. The day we'll need to do the same for another library, we'll be blocked.

Reply:
I've change it with a lazy for the tracker instanciation, thanks for the tip